ai-server/app/ml/train_enhanced_model.py

```
@@ -12,119 +15,143 @@
-    # 특성과 라벨 분리
-    # features 컬럼은 리스트 형태의 문자열이므로 파싱 필요
-    X = []
-    y = df['label'].values
+    # === 2. 특성과 라벨 분리 ===
+    # 특성(X): 판단 기준 (현재는 기술스택 겹치는 비율만 사용)
+    # 라벨(y): 정답 (추천해야 하는가? 1=예, 0=아니오)
+    X = []  # 특성 배열 (입력)
+    y = df['label'].values  # 라벨 배열 (출력/정답)
     
+    # CSV에서 특성 데이터를 문자열로 저장했으므로 숫자로 변환 필요
     for i, row in df.iterrows():
```

Ruff (0.12.2)

Loop control variable `i` not used within loop body

(B007)